src/scripts/main.js

```diff
@@ -1,3 +1,428 @@
+  }
+
+  row.setAttribute('class', 'active');
+});
+
+// #endregion
+
+// #region messeges
+
+const error = (message) => {
+  const notification = document.createElement('div');
+
+  notification.setAttribute('data-qa', 'notification');
+  notification.setAttribute('class', 'notification error');
+
+  const title = document.createElement('h2');
+
+  title.textContent = 'Error!';
+
+  const description = document.createElement('p');
+
+  description.textContent = message;
+
+  notification.append(title);
+  notification.append(description);
+
+  document.body.append(notification);
+};
+
+const success = () => {
+  const errorMessage = document.querySelector('.error');
+
+  if (errorMessage) {
+    errorMessage.remove();
+  }
+
+  const notification = document.createElement('div');
+
+  notification.setAttribute('data-qa', 'notification');
+  notification.setAttribute('class', 'notification success');
+
+  const title = document.createElement('h2');
+
+  title.textContent = 'Success!';
+
+  const description = document.createElement('p');
+
+  description.textContent = 'New employee added with success!';
+
+  notification.append(title);
+  notification.append(description);
+
+  document.body.append(notification);
+};
+
+// #endregion
+
+// #region converters
+const salaryConvertValue = (value) => {
+  const formattedNum = value.toLocaleString('en-US');
+
+  return '$' + String(formattedNum);
+};
+
+const wordConvert = (word) => {
+  return word.trim().charAt(0).toUpperCase() + word.slice(1).toLowerCase();
+};
+
+// #endregion
+
+// #region validationFunc
+const validation = () => {
+  const employeeName = document.querySelector("[data-qa='name']").value;
+  const position = document.querySelector("[data-qa='position']").value;
+  const office = document.querySelector("[data-qa='office']").value;
+  const age = Number(document.querySelector("[data-qa='age']").value);
+  const salary = Number(document.querySelector("[data-qa='salary']").value);
+
+  if (employeeName.trim().length < 4) {
+    const message = 'Name has less than 4 letters';
+
+    return error(message);
+  } else {
+    newEmployee.name = wordConvert(employeeName);
+  }
+
+  if (position.trim().length < 4) {
+    const message = 'Position has less than 4 letters';
+
+    return error(message);
+  } else {
+    newEmployee.position = position;
+  }
+
+  newEmployee.office = office;
+
+  if (age < 18 || age > 90) {
+    const message = 'Age is less than 18 or more than 90';
+
+    return error(message);
+  } else {
+    newEmployee.age = age;
+  }
+
+  newEmployee.salary = salaryConvertValue(salary);
+
+  return true;
+};
+// #endregion
+
+// #region submitFunc
+
+const addEmployee = () => {
+  const tr = document.createElement('tr');
+
+  if (validation()) {
+    for (const key in newEmployee) {
+      const td = document.createElement('td');
+
+      td.textContent = newEmployee[key];
+
+      tr.append(td);
+    }
+
+    tbody.append(tr);
+
+    reset();
+    success();
+  }
```

Comment:
The `addEmployee` function should handle the case where validation fails. Currently, if validation returns `false`, the function continues execution, which may lead to unexpected behavior. Consider adding a return statement after the validation check to exit the function early if validation fails.